README.md

```diff
@@ -58,8 +58,11 @@ docker ls
 docker run -it \
     -p 9091:80 \
     -e "API_URL=http://api.docker:7081" \
+    -e "AUTH0_DOMAIN=sil-appbuilder.auth0.com" \
+    -e "AUTH0_CLIENT_ID=n8IAE2O17FBrlQ667x5mydhpqelCBUWG" \
+    -e "DWKIT_UI_HOST=172.18.0.1" \
```

Comment:
this IP is probably machine specific, but nginx doesn't start without this variable being set